src/routes/trade/trade.module.scss

```diff
@@ -51,8 +71,14 @@
     cursor: pointer;
   }
 
+  &__button {
+    background-color: var(--section-background-1);
+    color: var(--text-main);
+    font-weight: bold;
+  }
   &--selected {
-    background-color: #939797;
+    background-color: #d7dadb;
```

Comment:
can we use the same color in `theme.scss`?

Reply:
updated